core/conf/conf_test.go

```diff
@@ -0,0 +1,54 @@
+package conf
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func setUpConfig(name, content string) {
+
+	ioutil.WriteFile(filepath.Join(getConfigDir(), name), []byte(content), 0744)
+}
+
+func tearDownConfig(name string) {
+	os.Remove(filepath.Join(getConfigDir(), name))
+}
+
+func TestMissingConfig(t *testing.T) {
```

Comment:
@cnlangzi Pls follow the unit-test format from 0chain repo or 0blobber. It's enough to have one function and inside have several tests, you just define subtests lile
`t.Run(YOUR TEST NAME`)